packages/shared/src/features/common/components/FormInputRating.tsx

```diff
@@ -66,11 +71,25 @@ export const FormInputRating = ({
               type="button"
               variant={ButtonVariant.Quiz}
             >
-              {item.label}
+              {index + 1}
             </Button>
           );
         })}
       </div>
+      <div className="flex flex-row justify-between">
+        <Typography
+          type={TypographyType.Callout}
+          color={TypographyColor.Tertiary}
+        >
+          {options.at(0)?.label}
+        </Typography>
+        <Typography
+          type={TypographyType.Callout}
+          color={TypographyColor.Tertiary}
+        >
+          {options.at(-1)?.label}
```

Comment:
What's the value at -1? Isn't `undefined`?

Reply:
-1 is the last value in array so we should have it since options are mandatory ✔️